dscript/commands/train.py

```diff
@@ -150,9 +155,31 @@ def add_args(parser):
         help="weight on the similarity objective (default: 0.35)",
     )
 
+    # Topsy-Turvy
+    train_grp.add_argument(
+        "--topsy-turvy",
+        dest="run_tt",
+        action="store_true",
+        help="run in Topsy-Turvy mode -- use top-down GLIDER scoring to guide training",
+    )
+    train_grp.add_argument(
+        "--glider-weight",
+        dest="glider_weight",
+        type=float,
+        default=0.2,
+        help="weight on the GLIDER accuracy objective (default: 0.2)",
+    )
+    train_grp.add_argument(
+        "--glider-thresh",
+        dest="glider_thresh",
+        type=float,
+        default=0.925,
+        help="proportion of GLIDER scores treated as positive edges (0 < gt < 1) (default: 0.925)",
```

Comment:
change ""proportion of GLIDER scores treated as positive edges (0 < gt < 1) (default: 0.925)""
to 
""threshold beyond which GLIDER scores treated as positive edges (0 < gt < 1) (default: 0.925)"

Reply:
Updated in 1b598e47620f313ebba9610154a87ee5785bf16d